evap/staff/tests/test_views.py

```diff
@@ -1101,8 +1101,9 @@ def test_publish_notifications(self):
         mail.outbox = []
 
         self.helper_publish_evaluation_with_publish_notifications_for(evaluation, contributors=True, participants=False)
-        self.assertEqual(len(mail.outbox), 1)
-        self.assertEqual(mail.outbox[0].to, [contributor1.email])
+        self.assertEqual(len(mail.outbox), 2)
+        self.assertIn([contributor1.email], [mail.outbox[0].to, mail.outbox[1].to])
+        self.assertIn([self.responsible.email], [mail.outbox[0].to, mail.outbox[1].to])
```

Comment:
Could these three lines become something like this?
I’d prefer `assertCountEqual` over multiple asserts as it doesn’t stop at the first error and gives a nice summary, while keeping the bag-of-lists-semantic.

```python
self.assertCountEqual([[contributor1.email], [self.responsible.email]], [email.to for email in mail.outbox])

Reply:
nice, wasn't aware of that one.